packages/transformer/src/IModelExporter.ts

```diff
@@ -1027,6 +1052,26 @@ export class ChangedInstanceOps {
         val.delete.forEach((id: Id64String) => this.deleteIds.add(id));
     }
   }
+
+  public get isEmpty(): boolean {
```

Comment:
Tried to use this from exporter and have suggestion: since `Exporter.sourceDbChanges` property is optional, I think `isEmpty` should be refactored into `hasChanges`, then instead of this
```
const isEmpty = this.exporter.sourceDbChanges === undefined || this.exporter.sourceDbChanges.isEmpty;
const hasChanges = this.exporter.sourceDbChanges !== undefined && !this.exporter.sourceDbChanges.isEmpty
```
we could do this:
```
const isEmpty = !this.exporter.sourceDbChanges?.hasChanges;
const hasChanges = this.exporter.sourceDbChanges?.hasChanges
```

Another solution could be to make `Exporter.sourceDbChanges` required, but this seems like a lot more work.

Reply:
changed to hasChanges